viewer/packages/tools/src/Overlay3D/Overlay3DTool.ts

```diff
@@ -44,6 +44,26 @@ export type Overlay3DToolParameters = {
   defaultOverlayColor: THREE.Color;
 };
 
+export type OverlayCollectionOptions = {
+  /**
+   * Sets default overlay color for newly added overlays.
+   * Default is yellow.
+   * */
+  defaultOverlayColor?: THREE.Color;
+  /**
+   * Sets default texture for all overlays of this OverlayCollection.
+   * Must be a square texture with size at least as `maxPointSize`.
```

Comment:
Why does it have to be of size at least maxPointSize?

Reply:
No specific reason, just because it won't look pixelated if it is at least `maxPointSize`. Can just rephrase it as recommendation.